### DISCUSSION TITLE
Pseudonym support for collaborative fiction community

### DISCUSSION THREAD

**Discussion by BruceCaron** [knowledgefutures/pubpub#1843] [Ideas]
I'm building a community around a theory fiction premise. 

I want to invite authors to collaborate.  They can embellish the premise and they can add their own stories.

I am looking to attract authors of note, who might prefer to use a pseudonym for this work. 

Is it possible to have support for pseudonyms... even if the author must include their actual name to sign into pubpub?

**Comment by gabestein** [comment]
Hi Bruce — sounds like a super cool project! Yes, this is possible, as Pub access control ("members") is separate from the contributor/author listing on a Pub, and you can add a contributor with any name you like. So all you'd need to do is tell authors to remove their accounts from the contributor listing, and add the pseudonymous author, like so:

<img width="1023" alt="Screen Shot 2022-03-02 at 11 13 29" src="https://user-images.githubusercontent.com/639110/156401893-2bce0bf5-32e0-4665-b9aa-692f155d1d6c.png">

Here's an example of the finished result: https://demo.pubpub.org/pub/bls5q15c/release/1

**Comment by BruceCaron** [comment]
Great! I love a feature request that uses an existing feature.

b

On Wed, Mar 2, 2022 at 8:16 AM Gabriel Stein ***@***.***>
wrote:

> Hi Bruce — sounds like a super cool project! Yes, this is possible, as Pub
> access control ("members") is separate from the contributor/author listing
> on a Pub, and you can add a contributor with any name you like. So all
> you'd need to do is tell authors to remove their accounts from the
> contributor listing, and add the pseudonymous author, like so:
>
> [image: Screen Shot 2022-03-02 at 11 13 29]
> <https://user-images.githubusercontent.com/639110/156401893-2bce0bf5-32e0-4665-b9aa-692f155d1d6c.png>
>
> Here's an example of the finished result:
> https://demo.pubpub.org/pub/bls5q15c/release/1
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pubpub/pubpub/discussions/1843#discussioncomment-2281634>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA5QY6ARE4Q6ERHIAZGRHKLU56H4HANCNFSM5PU7OXSQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Bruce Caron, PhD
New Media Studio, 501 (c) 3
417 Samarkand Drive
Santa Barbara, CA 93105
email me for a phone number...
